lib/ansible/modules/cloud/podman/podman_image.py

```diff
@@ -0,0 +1,720 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = """
+  module: podman_image
+  author:
+      - Sam Doran (@samdoran)
+  version_added: '2.8'
+  short_description: Pull images for use by podman
+  notes: []
+  description:
+      - Build, pull, or push images using Podman.
+  options:
+    name:
+      description:
+        - Name of the image to pull, push, or delete. It may contain a tag using the format C(image:tag).
+      required: True
+    executable:
+      description:
+        - Path to C(podman) executable if it is not in the C($PATH) on the machine running C(podman)
+      default: 'podman'
+      type: str
+    cert_dir:
+      description:
+        - Path to directory containing TLS certificates and keys to use
+      type: 'path'
+    tag:
+      description:
+        - Tag of the image to pull, push, or delete.
+      default: "latest"
+    pull:
+      description: Whether or not to pull the image.
+      default: True
+    push:
+      description: Whether or not to push an image.
+      default: False
+    path:
+      description: Path to directory containing the build file.
+    force:
+      description:
+        - Whether or not to force push or pull an image. When building, force the build even if the image already exists.
+    state:
+      description:
+        - Whether an image should be present, absent, or built.
+      default: "present"
+      choices:
+        - present
+        - absent
+        - build
+    tls_verify:
```

Comment:
I'd rather see this be `validate_certs` with `tls_verify` as an alias (for those who want to maintain the `podman` vernacular), but favor the ansible core default of `validate_certs` by default.

Reply:
Will do.